packages/api/amplify_api_android/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/NativeApiPluginBindings.java

```diff
@@ -1,7 +1,7 @@
 // 
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-// Autogenerated from Pigeon (v7.1.5), do not edit directly.
+// Autogenerated from Pigeon (v9.2.4), do not edit directly.
```

Comment:
Files that end in `Pigeon.java`or `Messages.java` will be treated as generated. We could rename this, but this should be removed soon anyway, so not really a big deal.